src/digest.rs

```diff
@@ -471,6 +528,22 @@ mod tests {
     use std::vec::Vec;
     use super::super::{digest, file_test};
 
+    #[test]
+    fn test_sha1_sizes() {
+        use core::mem::size_of;
+        use super::{Sha1State, Sha1Block};
+
+        assert_eq!(digest::SHA1.block_len, size_of::<Sha1Block>());
+        assert_eq!(digest::SHA1.chaining_len, size_of::<Sha1State>());
+    }
+
+    #[test]
+    fn test_single_sha1() {
```

Comment:
Removed. (I’d included this to be able to trace through a single digest for debugging, but that bug is solved now.)